packaging/debian/build_debian.sh

```diff
@@ -52,6 +64,11 @@ build_one_architecture() {
     > "$dest/build/DEBIAN/control"
   cp "$here/postinst" "$dest/build/DEBIAN/"
 
+  # Now the Electron build.
+  echo "Building Electron client for $electron_arch"
+  (cd ../../desktop && node package.js --platform linux --arch $electron_arch)
+  (cd ../../desktop && rsync -a release/linux-${electron_arch}/Keybase-linux-${electron_arch}/ "$dest/build/opt/keybase")
```

Comment:
I like to quote all variable expansions as a matter of religious faith, but if you prefer not to with constants-known-not-to-contain-spaces I'm ok with that.